### PR TITLE
Adding host list dos and tags

### PR DESCRIPTION
**Pull request by stephanwampouille**
closes: #615

### CODE REVIEW

File: apps/detect/src/components/hosts/host-list.tsx [hunk 1]
```diff
@@ -20,6 +30,7 @@ const HostList: React.FC<{ data: ProbeActivity[] }> = ({ data }) => {
         <HostRow
           key={e.endpoint_id}
           endpoint={e}
+          platform={e.dos}
```
Comment (kenrick) [inline apps/detect/src/components/hosts/host-list.tsx:33]: we pass the entire `endpoint` as a prop. can we get the `dos` from that instead

File: apps/detect/src/components/hosts/host-list.tsx [hunk 2]
```diff
@@ -60,6 +72,17 @@ const HostRow: React.FC<{
...
 
         <div className={styles.details}>
+          {platform && (
+            <HostDetail
+              icon={<VariantIcon platform={platform.split("-")[0]} />}
+              comment={platform.split("-")[0]}
+            />
+          )}
+          {probe?.tags &&
+            probe.tags.map(
```
Comment (kenrick) [inline apps/detect/src/components/hosts/host-list.tsx:82]: I think this should do the same thing
```suggestion
          {probe?.tags?.map(
```

File: apps/detect/src/components/hosts/host-list.tsx [hunk 3]
```diff
@@ -60,6 +72,17 @@ const HostRow: React.FC<{
...
+              (tag) =>
+                tag !== "" && <HostDetail icon={<TagIcon />} comment={tag} />
```
Comment (kenrick) [inline apps/detect/src/components/hosts/host-list.tsx:84]: I would filter out this tag with

```
probe?.tags?.filter(tag  => tag !== "").map(...

```

**Review by kenrick — CHANGES_REQUESTED** [review]

File: apps/detect/src/components/hosts/host-list.tsx (hunk 1 above)
Comment (stephanwampouille) [reply apps/detect/src/components/hosts/host-list.tsx:33]: oh yup lol

**Review by kenrick — APPROVED** [review]

**Pull request merged**